hardware/Layout - desing- guide/Layout-Desing-Guide.md

```diff
@@ -0,0 +1,464 @@
+
```

Comment:
Add a Table of Contents at beginning of the doc.  Using HTML tag to fold/unfold it. Like this one
```
<details>
<summary><font size=5>Table of Contents</font> </summary>
- [1. Introduction](#1-introduction)
    - [1.1. Application features](#11-application-features)
    - [1.2. Purpose](#12-purpose)
- [2. Fundamental steps](#2-fundamental-steps)
...
</details>
```

Reply:
Done